indico/modules/events/contributions/util.py

```diff
@@ -72,3 +78,80 @@ def serialize_contribution_person_link(person_link):
     data['authorType'] = person_link.author_type.value
     data['isSubmitter'] = person_link.is_submitter
     return data
+
+
+class ContributionReporter(ReporterBase):
+    """Class performing reporting and filtering actions in the contribution
+    report
+    """
+    CSRF_ENABLED = True
+    REPORT_ID = 'contribution_report'
+    DEFAULT_REPORT_CONFIG = {'filters': {'items': {}}}
+    FILTERABLE_ITEMS = OrderedDict([
+        ('session', {
+            'title': _('Session'),
+            'id': 'session',
```

Comment:
I can adapt the UI where I render the filter options so that instead of `itervalues()` I'll use `iteritems()`. Should have done that from the beginning.